Bigtable/tests/Conformance/proxy/known_failures.txt

```diff
@@ -0,0 +1,10 @@
+_Retry_WithRoutingCookie
+_Retry_WithRetryInfo
+TestMutateRows_Generic_DeadlineExceeded
+TestReadRow_Generic_DeadlineExceeded
+TestReadRows_NoRetry_OutOfOrderError
```

Comment:
Can we remove TestReadRows_NoRetry_OutOfOrderError, TestReadRows_Retry_StreamReset, TestReadRows_Retry_PausedScan and TestReadRows_Retry_LastScannedRow from this list? So we don't forget about them :) We don't have to mark conformance test passing as a required step for presubmit for now.

Reply:
I would personally rather skip these and have them passing (in order to ensure no regression) than to have them fail and ignore the failing tests going forward.

I've entered bugs to fix these failing tests, which is a better place to keep track of these issues, so I don't think we'll forget them.